rpc/lib/server/http_server.go

```diff
@@ -102,15 +102,16 @@ func StartHTTPAndTLSServer(
 		listener = netutil.LimitListener(listener, config.MaxOpenConnections)
 	}
 
-	go func() {
```

Comment:
I will ask you to apply the same changes to `StartHTTPServer` func. We need to keep them consistent, so we can swap HTTP server with HTTPS.

Reply:
Will do shortly

Reply:
@melekes Done

Reply:
mmm tests failed, I'm on it